docker-build-registry/action.yml

```diff
@@ -0,0 +1,92 @@
+name: "Build and push docker"
+description: "Build and push docker image"
+
+inputs:
+  registry:
+    description: "Image registry host"
+    required: true
+  public_registry_prefix:
+    description: "Extra public registry host"
+    default: "ghcr.io/espoon-voltti"
+
+  name:
+    description: "Name of the image. Example evaka/image"
+    required: true
+  cache_from:
+    description: "Get cache from this image."
+    default: ""
+    required: false
+  path:
+    description: "Docker context path"
+    default: "."
+    required: false
+  build-args:
+    description: "Docker build-args. See docker/build-push-action build-args"
+    required: false
+    default: ""
+  target:
+    description: "Docker target"
+    default: null
+  push:
+    description: "Push images to registry"
+    default: true
+  load:
+    description: "Load docker image"
+    default: false
+  dockerfile:
+    description: "Dockerfile path"
+  no_sha:
+    default: false
+
+outputs:
```

Comment:
Could have output variables as:
* image_full_name # registry + image + tag
* image_name
* image_registry
* image_tag

Reply:
Changed so ecr_registry is exported from setup instead here. The image_tag can have multiple values. image_full_name can be confuced to name:tag. instead of registry/name:tag. I hope one day they actually give real names for each variant.

Reply:
True, image_fqn (fully qualified name) or or image_fqin (fully qualified image name) are the correct terms.

Reply:
Could not find info about FQN. FQIN has problem that is has optional tag portion, so it is not really exact either in this case.